[AOTI] Forward fix a Windows build failure

desertfire opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack) (oldest at bottom):
* __->__ #116790

Summary: forward fix https://github.com/pytorch/pytorch/pull/116269

desertfire committed:
[AOTI] Forward fix a Windows build failure

Summary: forward fix https://github.com/pytorch/pytorch/pull/116269

[ghstack-poisoned]

desertfire committed:
Update on "[AOTI] Forward fix a Windows build failure"


Summary: forward fix https://github.com/pytorch/pytorch/pull/116269

[ghstack-poisoned]

desertfire committed:
Update on "[AOTI] Forward fix a Windows build failure"


Summary: forward fix https://github.com/pytorch/pytorch/pull/116269

[ghstack-poisoned]

huydhn reviewed (approved):
Thank you for the fix!

desertfire committed:
Update on "[AOTI] Forward fix a Windows build failure"


Summary: forward fix https://github.com/pytorch/pytorch/pull/116269

[ghstack-poisoned]

desertfire commented:
@pytorchbot merge
